### DISCUSSION TITLE
Horizon balancing issues

### DISCUSSION THREAD

**Discussion by weblogjacob** [laravel/framework#35800] [Q&A]
Since upgrading to Laravel 8 and Horizon 5.6 we are seeing balancing issues with horizon. Queues with a lot of jobs do not get workers. On the contrary, these queues tend to have 0 workers. What could be the problem?

(Redacted) config:

```
'environments' => [
    'production' => [
        'supervisor-1' => [
            'connection' => 'redis-long-running',
            'queue' => [
                '...',
                '...',
            ],
            'balance' => 'auto',
            'minProcesses' => 1,
            'maxProcesses' => 26,
            'balanceMaxShift' => 3,
            'balanceCooldown' => 15,
            'memory' => 512,
            'tries' => 1,
            'timeout' => 3600,
        ],
    ],
],
```

![Schermafbeelding 2021-01-06 om 15 09 49](https://user-images.githubusercontent.com/73335453/103777923-e0cbbc00-5031-11eb-83a6-86ca5ba09ac3.png)

**Comment by weblogjacob** [comment]
Anyone experiencing the same problem?

**Reply by myckhel** [reply]
it's better you open an issue otherwise i don't think you will get attention here.
The laravel team has not shown enough encouragement towards using the discussion feature.
